app/scripts/lib/metrics.js

```diff
@@ -121,6 +122,7 @@ define([
     this._env = options.environment || new Environment(this._window);
 
     this._lastAbLength = 0;
+    this._startTime = options.startTime || Date.now();
```

Comment:
The event times returned by SpeedTrap are based on a start time it keeps internally. You can use it's start time like so:

```js
this._startTime = options.startTime || this._speedTrap.baseTime;
```